src/database/migrations/20240801074543-merge-mentor-user-extensions.js

```diff
@@ -0,0 +1,155 @@
+'use strict'
+require('module-alias/register')
+require('dotenv').config()
+const materializedViewsService = require('@generics/materializedViews')
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.sequelize.query('DROP MATERIALIZED VIEW IF EXISTS m_user_extensions;')
+		await queryInterface.sequelize.query('DROP MATERIALIZED VIEW IF EXISTS m_mentor_extensions;')
+
+		await queryInterface.sequelize.transaction(async (transaction) => {
+			// Add the is_mentor flag to user_extensions
+			await queryInterface.addColumn(
+				'user_extensions',
+				'is_mentor',
+				{
+					type: Sequelize.BOOLEAN,
+					allowNull: false,
+					defaultValue: false,
+				},
+				{ transaction }
+			)
+
+			// Copy data from mentor_extensions to user_extensions
+			await queryInterface.sequelize.query(
+				`
+        INSERT INTO user_extensions (
+          user_id, designation, area_of_expertise, education_qualification,
+          rating, meta, stats, tags, configs, mentor_visibility, visible_to_organizations,
+          external_session_visibility, external_mentor_visibility, custom_entity_text,
+          experience, organization_id, external_mentee_visibility, mentee_visibility,
+          name, email, phone, is_mentor,created_at,updated_at
+        )
+        SELECT
+          user_id, designation, area_of_expertise, education_qualification,
+          rating, meta, stats, tags, configs, mentor_visibility, visible_to_organizations,
+          external_session_visibility, external_mentor_visibility, custom_entity_text,
+          experience, organization_id, external_mentee_visibility, mentee_visibility,
+          name, email, phone, true,created_at,updated_at
+        FROM mentor_extensions
+      `,
+				{ transaction }
+			)
+
+			// Drop the mentor_extensions table
+			await queryInterface.dropTable('mentor_extensions', { transaction })
+		})
+		await materializedViewsService.checkAndCreateMaterializedViews()
+	},
+
+	down: async (queryInterface, Sequelize) => {
```

Comment:
@nevil-mathew mentor_extensions view creation is missing

Reply:
`await materializedViewsService.checkAndCreateMaterializedViews()`

This function will create all the necessary view. Also, we don't need mentor view since it's table is removed.

Reply:
in down we don't have it, so can you add it

Reply:
added